superset/assets/src/dashboard/components/Header.jsx

```diff
@@ -314,6 +315,9 @@ class Header extends React.PureComponent {
               isStarred={this.props.isStarred}
             />
           </span>
+          <span className="dashboard-loading">
```

Comment:
it might be cleaner to wrap the span with the boolean so it doesn't even render when it's not loading